hack/vendor.sh

```diff
@@ -29,7 +29,7 @@ clone git github.com/RackSec/srslog 6eb773f331e46fbba8eecb8e794e635e75fc04de
 clone git github.com/imdario/mergo 0.2.1
 
 #get libnetwork packages
-clone git github.com/docker/libnetwork v0.6.1-rc2
+clone git github.com/docker/libnetwork v0.7.0-dev.2
```

Comment:
correct. I will push a individual patch in 1.10.1 that fixes https://github.com/docker/docker/issues/20140 alone. Since it is a P3 and not specific to 1.10 (it was seen also in 1.9), we can decide to pick it up or not, based on priority and release requirements.